docs/components/gantry/multiaxis.md

```diff
@@ -0,0 +1,219 @@
+---
+title: "Configure a Multi-Axis Gantry"
+linkTitle: "multiaxis"
+weight: 80
+type: "docs"
+description: "Configure a multiaxis gantry."
+tags: ["gantry", "components"]
+# SME: Rand
+---
+
+Configure a `multiaxis` gantry to integrate a multi-axis gantry into your robot:
+
+{{< tabs >}}
+{{% tab name="Config Builder" %}}
+
+Navigate to the **config** tab of your robot's page in [the Viam app](https://app.viam.com).
+Click on the **Components** subtab and navigate to the **Create component** menu.
+Enter a name for your gantry, select the type `gantry`, and select the `multiaxis` model.
+
+Click **Create component**.
+Paste into the **Attributes** box:
+
+``` json
+"attributes": {
+    "subaxes_list": [
+        <"xaxis-name">,
+        <"yaxis-name">,
+        <"zaxis-name">
+    ]
+}
+```
+
+![Creation of a multi-axis gantry component in the Viam app config builder.](../img/multiaxis-ui-config.png)
+
+Edit and fill in the attributes as applicable.
+
+{{% /tab %}}
+{{% tab name="JSON Template" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "components": [
+    // your motor & oneaxis gantry configs
+    {
+        "name": <"your-fake-gantry-name">,
+        "type": "gantry",
+        "model": "multiaxis",
+        "attributes": {
+            "subaxes_list": [
+                <"xaxis-name">,
+                <"yaxis-name">,
+                <"zaxis-name">
+            ]
+        },
+      "depends_on": []
+    }
+  ]
+}
+```
+
+{{% /tab %}}
+{{% tab name="JSON Example" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+    "components": [
+        {
+            "name": "local",
+            "type": "board",
+            "model": "pi"
+        },
+        {
+            "name": "xmotor",
+            "type": "motor",
+            "model": "gpiostepper",
+            "attributes": {
+                "board": "local",
+                "pins": {
+                    "dir": "dirx",
+                    "pwm": "pwmx",
+                    "step": "stepx"
```

Comment:
```suggestion
                    "dir": "11",
                    "pwm": "13",
                    "step": "15"
```
This is I think meant to be a working config and it won't work without pin numbers